packages/fillings/src/createRuntimeFillings.ts

```diff
@@ -0,0 +1,123 @@
+import { assignInlineVars } from '@vanilla-extract/dynamic';
+
+import type {
+  BaseConditions,
+  ConditionalFillingsConfig,
+  CSSVarFunction,
+  FillingsConfig,
+  FillingsFn,
+  ComputeFillingsProps,
+} from './types';
+
+export function createRuntimeFillings<
+  Config extends ConditionalFillingsConfig<BaseConditions, string>,
+>(config: Config): FillingsFn<Config>;
+export function createRuntimeFillings<Config extends FillingsConfig<string>>(
+  config: Config,
+): FillingsFn<Config>;
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function createRuntimeFillings(config: any): any {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  return (options: any) => {
+    if (typeof options !== 'object' || options === null) {
+      return {
+        className: '',
+        assignVars() {
```

Comment:
What's the reasoning for `assignVars` being a function? Seeing as it doesn't ever take any parameters I would've thought it'd be better represented as `style` seeing as it's effectively an inline style object.

Reply:
Guess I wanted to mimic the API of `dynamic` library, but you're right we will be always calling this one at the end of the day. Will refactor it to a `style` object